config/locales/emails/candidate_mailer.yml

```
@@ -6,6 +6,16 @@ en:
       subject: "You’ve submitted your teacher training application"
     application_submitted_apply_again:
       subject: "You’ve submitted your teacher training application"
+    candidate_offer:
+      single_offer:
+        subject: "Make a decision: successful application for %{provider_name}"
+      multiple_offers:
+        subject: "Make a decision: successful application for %{provider_name}"
+      decisions_pending:
+        subject: "Successful application for %{provider_name}"
+    chase_candidate_decision_email:
```

`email` seems redundant here

```suggestion
    chase_candidate_decision:
```